File: frontend/fin-tech/src/models/NaturezaLancamento.js
export default class NaturezaLancamento {
  constructor(obj) {
    obj = obj || {};

    this.id = obj.id;
    this.idUsuario = obj.idUsuario;
    this.descricao = obj.descricao || null;
    this.observacao = obj.observacao || null;
    this.dataCadastro = obj.dataCadastro || null;
  }
}

File: backend/src/FinTech.Api/Domain/Models/NaturezaLancamento.cs
using System.ComponentModel.DataAnnotations;

namespace FinTech.Api.Domain.Models
{
    public class NaturezaLancamento
    {
        [Key]
        public long Id { get; set; }

        [Required]
        public long IdUsuario { get; set; }

        public Usuario Usuario { get; set; }

        [Required(ErrorMessage = "O campo de Descrição é obrigatório.")]
        public string Descricao { get; set; }

        public string? Observacao { get; set; } = string.Empty;

        [Required]
        public DateTime DataCadastro { get; set; }
        
    }
}